api/toc.yml

```diff
@@ -1,6 +1,6 @@
-- name: Reference
+- name: API reference
```

Comment:
I like disambiguating this way, but I am slightly concerned with the accessibility of the term "API." Do we have data on how familiar new developers or non-developers are with the term?

Reply:
Great questions. Would new developers/non-developers want to look at the references anyways, though? Are there alternative names to use?

Reply:
I expect they would, but I'll admit I don't have data to back that up. Maybe @bromeg or @natke, do you have any data available here?

Reply:
I'd say disambiguation is the most important part, especially with "reference" being such a generic term on its own. Honestly I think that even if non-devs aren't familiar with the term, with a single click into the section they'll immediately be saying "oh that's what API references are, okay cool" and voila no more worries. 

Other options _might_ be "Code reference" or "Language reference", but I think nerfing the term to something less specific is only inviting confusion.

Reply:
Fair enough! If we do add a language reference as a distinct header, then, it should likely go adjacent to the API reference header:

- API reference
  - Q# libraries
  - .NET libraries
  - Python libraries
- Q# language reference